docs/payment/index.md

```diff
@@ -0,0 +1,34 @@
+# Payment gateways configuration
+
+Saleor App Checkout supports two payment gateways that you can configure:
+
+<a href="https://www.mollie.com/en">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./docs/logos/mollie_light.svg">
+    <source media="(prefers-color-scheme: light)" srcset="./docs/logos/mollie_dark.svg">
+    <img alt="Mollie" src="./docs/logos/mollie_dark.svg">
+  </picture>
+</a>
+
+<br>
```

Comment:
I think we can leave that part in main README so that our users know which payment gateways we support out of the box.

@mmiszy We need to add Stripe here as well :D